src/ProjectNeon/Assets/Features/Battle/Cards/PlayedCard.cs

```diff
@@ -1,12 +1,29 @@
 ﻿using UnityEngine;
 
+/**
+ * Represents a played card from enemy or player character.
+ */
+ 
 public class PlayedCard : MonoBehaviour
 {
-    private Member performer;
-    private Target target;
+    /**
+     * Card played
+     */
     private Card card;
 
-    public PlayedCard Init(Member performer, Target target, Card card)
+    /**
+     * Member which played the card
+     */
+    private Member performer;
+
+    /**
+     * Each Target targeted by the Card effects in the same order that the effects are stored in Card.
+     * 
+     * @todo #207:30min We need a better solution to link targets to effects.
```

Comment:
This puzzle isn't clear. Let's use clear and unambiguous language about what the exact problem is.

Reply:
@SilasReinagel I think that it's better now, please take a look and say what yo think